eng/pipelines/common/xplat-setup.yml

```diff
@@ -126,12 +126,12 @@ jobs:
         # Official Build Windows Pool
         ${{ if and(eq(parameters.osGroup, 'windows'), ne(variables['System.TeamProject'], 'public')) }}:
           name: NetCore1ESPool-Svc-Internal
-          demands: ImageOverride -equals Build.Windows.10.Amd64.VS2019
+          demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Pre
```

Comment:
Is it expected that we have to use VS2022 **Preview** to build the stable 6.0 LTS servicing branches?  That seems pretty wrong to me.  We try to avoid churning the build tools in servicing.

Reply:
Yeah, I think we actually want regular VS2022 here.

Reply:
Is that going to cause an observable change to dependencies of native code-- moving from VS2019 to VS2022 in servicing?

Reply:
From our conversation on chat: I'll revert the last commit, and then Alexander will take a look at the necessary changes in the llvm project.

Reply:
We'll go back to VS2019 in llvm-project.

Reply:
Reverted. Next step: Wait for darc to flow the llvm changes into this PR.